src/planscape/climate_foresight/tasks.py

```diff
@@ -581,8 +577,8 @@ def process_landscape_datalayers(result: dict) -> dict:
         f"Processing landscape datalayers: current={current_id}, future={future_id}"
     )
 
-    datalayer_uploaded(current_id)
-    datalayer_uploaded(future_id)
+    datalayer_uploaded.delay(current_id)
```

Comment:
Soooo quick comment about triggering tasks from other tasks...

While it's usually ok to do so, Celery is a pain and sometimes does not like it.

Older versions would crash n burn terribly because of this and memory leaks sprung everywhere. Please be aware and monitor this.